DISCOVER/_config.yml

```
@@ -21,3 +21,11 @@ repository:
 html:
   use_issues_button: true
   use_repository_button: true
+
+# config sphinx tags extension
+sphinx:
+  extra_extensions:
+   - sphinx_tags
+  config:
+    tags_create_tags: True
+    tags_extension: ["md"]  # using MyST, not default: ["rst"]
```

```suggestion
    tags_extension: ["md"]  # using MyST, not default: ["rst"]
    suppress_warnings: ["etoc.toctree"]
```

This prevents the `WARNING: toctree directive not expected with external-toc [etoc.toctree]` (see melissawm/sphinx-tags#48)